react/features/app/components/App.native.js

```diff
@@ -105,6 +111,21 @@ export class App extends AbstractApp {
                 }
             }
 
+            // SONRISA MODIFICATION: handle config set by native app
```

Comment:
I don't think it's needed to state this is a sonrisa modification, it'll be clear when doing a blame